lib/project.ts

```diff
@@ -51,6 +51,11 @@ export class Project extends ProjectBase implements Project.IProject {
 				"The AppBuilder CLI lets you target only Apache Cordova 3.0.0 or later. " +
 				"To develop your projects with Apache Cordova 2.x, run the AppBuilder Windows client or the in-browser client.");
 		}
+
+		if (this.projectData && this.projectData.Framework) {
+			this.frameworkProject = this.$frameworkProjectResolver.resolve(this.projectData.Framework);
```

Comment:
we must make `frameworkProject` to has only getter, no need to set it on several places in this file. However this should be handled in separate PR